### DISCUSSION TITLE
Not good: If repo2 ever uses repo1, repo1 will forever show repo2 under "used by"

### DISCUSSION THREAD

**Discussion by jtlapp** [community/community#62080] [Repositories]
### Select Topic Area

Product Feedback

### Body

I informed GitHub support that a repo my app has not used in six months is still listing my app under "used by." Here is GitHub's response:

> This is expected behavior at this time. The Dependency graph will show code dependents if a repository has ever listed that dependency in a manifest file, not just if it is listed in the current one.

This makes me feel ill, because I tried that repo and did not like the approach, but now Github is forever associating my name and app with that repo.

I'm half inclined to purge my repo and start over.

**Comment by port19x** [comment]
I suggest you use this to troll github and "use" community/community.
Otherwise this will likely stay an open issue for ages
